packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.test.tsx

```diff
@@ -174,7 +174,7 @@ describe('VitalsBiometricsForm: ', () => {
       message: 'Internal Server Error',
       response: {
         status: 500,
-        statusText: 'Internal Server Error',
+        statusText: 'Some of the values entered are invalid',
```

Comment:
This should also use the translation
```suggestion
        statusText: t('checkForValidity', 'Some of the values entered are invalid'),
```